sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -582,12 +387,12 @@ namespace Azure { namespace Core { namespace Http {
     /**
      * @brief Get URL.
      */
-    Url& GetUrl() { return this->m_url; }
+    Azure::Core::Internal::Http::Url& GetUrl() { return this->m_url; }
```

Comment:
We can't have a public API like `GetUrl()` return an internal-only type.

Reply:
Yes, I am still thinking how to hide Request from the public.

I am still thinking how to hide the Request from public.  End-users should never create a Request.

The only expected case for end-users interacting with a request is when they create a custom policy where they are granted access to our implementation details directly.